Index: src/main/resources/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Spring application name\r\nspring.application.name=bbabap\r\n\r\n# Port\r\nserver.port=8880\r\n\r\n# DB\r\nspring.datasource.url=jdbc:mysql://localhost:3306/pabap?useSSL=false&serverTimezone=UTC\r\nspring.datasource.username=root\r\nspring.datasource.password=123456789\r\nspring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver\r\nspring.jpa.database-platform=org.hibernate.dialect.MySQLDialect\r\nspring.main.allow-circular-references=true\r\nlogging.level.root=DEBUG\r\n# ?? ?? ??\r\nlogging.level.com.encore.bbabap=DEBUG\r\n\r\n# ?? ?? ?? ??\r\nlogging.file.name=logs/app.log\r\n\r\nspring.security.enable-csrf=false\r\nspring.jpa.properties.hibernate.show_sql=true\r\nspring.jpa.properties.hibernate.use_sql_comments=true\r\nspring.jpa.properties.hibernate.format_sql=true\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>ISO-8859-1
===================================================================
diff --git a/src/main/resources/application.properties b/src/main/resources/application.properties
--- a/src/main/resources/application.properties	(revision 395198312aaf6b408fdb062d139d9a673eadb9ff)
+++ b/src/main/resources/application.properties	(date 1710872361449)
@@ -16,7 +16,7 @@
 logging.level.com.encore.bbabap=DEBUG
 
 # ?? ?? ?? ??
-logging.file.name=logs/app.log
+#logging.file.name=logs/app.log
 
 spring.security.enable-csrf=false
 spring.jpa.properties.hibernate.show_sql=true
